### PR TITLE
Attempting to edit a menu command results in an error

### PR DESCRIPTION
**Pull request by jgfoster**
The error was introduced by 361cecf8d43126f6bae62c61d66914c2ddf53c6f. This commit demonstrates the problem with a test, provides a fix, and confirms that the test passes. I fully expect that there is a better test and fix, but this seemed like a good way to communicate the problem and allows me to get beyond the walkback.

### CODE REVIEW

File: Core/Object Arts/Dolphin/IDE/PublishedAspectInspectorTest.cls [hunk 1]
```diff
@@ -240,6 +240,11 @@ testAspectChanged
 	self assert: treePresenter view itemCount == 1.
 	self repaint!
 
+testCommandMenuItemEditor
+	"#show reproduces the error and is less complicated to debug/test than #showModal"
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/IDE/PublishedAspectInspectorTest.cls]: This comment won't be clear to someone who doesn't have the context of the issue, especially as time goes on.

File: Core/Object Arts/Dolphin/IDE/PublishedAspectInspectorTest.cls [hunk 2]
```diff
@@ -702,6 +707,7 @@ testSetToNil
 !PublishedAspectInspectorTest categoriesFor: #tearDown!public!running! !
 !PublishedAspectInspectorTest categoriesFor: #testAddRemoveCollectionElement!public!unit tests! !
 !PublishedAspectInspectorTest categoriesFor: #testAspectChanged!public!unit tests! !
+!PublishedAspectInspectorTest categoriesFor: #testCommandMenuItemEditor!public! !
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/IDE/PublishedAspectInspectorTest.cls]: category

File: Core/Object Arts/Dolphin/MVP/Views/Common Controls/IconicListAbstract.cls [hunk 3]
```diff
@@ -293,6 +293,10 @@ getInfoTipBlock: anObject
 
 	getInfoTipBlock := anObject!
 
+hasGridLines: aBoolean
+	"Exists in ListView but not elsewhere. See
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/MVP/Views/Common Controls/IconicListAbstract.cls]: The method comment should be similar to that in ListView, e.g. "If supported, turn on or off the display of grid lines in the receiver depending on the value of the argument, aBoolean."
For completeness the following should also be defined:
```
hasGridLines
	"Answer whether the receiver is displaying gridlines."

	^false
```

File: Core/Object Arts/Dolphin/MVP/Views/Common Controls/IconicListAbstract.cls [hunk 4]
```diff
@@ -738,6 +742,7 @@ wmDestroy: message wParam: wParam lParam: lParam
 !IconicListAbstract categoriesFor: #getImageBlock:!adapters!public! !
 !IconicListAbstract categoriesFor: #getInfoTipBlock!accessing!public! !
 !IconicListAbstract categoriesFor: #getInfoTipBlock:!accessing!public! !
+!IconicListAbstract categoriesFor: #hasGridLines:!public! !
```
Comment (blairmcg) [inline Core/Object Arts/Dolphin/MVP/Views/Common Controls/IconicListAbstract.cls]: category

**Comment by jgfoster** [comment]
Thanks for making the needed changes!

**Pull request merged**